### ISSUE TITLE
🐛Cant use on Next js getting "Cannot read properties of null (reading 'length')" Error

### ISSUE THREAD

**Issue by cgtysnpr** [inovua/reactdatagrid#296]
- what edition are you using - community
- version for `@inovua/reactdatagrid-community`or `@inovua/reactdatagrid-enterprise` - ^4.21.0

Relevant code or config

```import ReactDataGrid from "@inovua/reactdatagrid-community";
import "@inovua/reactdatagrid-community/index.css";
const Main = () => {
  const columns = [
    { name: "name", header: "Name", minWidth: 50, defaultFlex: 2 },
    { name: "age", header: "Age", maxWidth: 1000, defaultFlex: 1 },
  ];
  const dataSource = [
    { id: 1, name: "John McQueen", age: 35 },
    { id: 2, name: "Mary Stones", age: 25 },
    { id: 3, name: "Robert Fil", age: 27 },
    { id: 4, name: "Roger Robson", age: 81 },
    { id: 5, name: "Billary Konwik", age: 18 },
    { id: 6, name: "Bob Martin", age: 18 },
    { id: 7, name: "Matthew Richardson", age: 54 },
    { id: 8, name: "Ritchie Peterson", age: 54 },
    { id: 9, name: "Bryan Martin", age: 40 },
    { id: 10, name: "Mark Martin", age: 44 },
    { id: 11, name: "Michelle Sebastian", age: 24 },
    { id: 12, name: "Michelle Sullivan", age: 61 },
    { id: 13, name: "Jordan Bike", age: 16 },
    { id: 14, name: "Nelson Ford", age: 34 },
    { id: 15, name: "Tim Cheap", age: 3 },
    { id: 16, name: "Robert Carlson", age: 31 },
    { id: 17, name: "Johny Perterson", age: 40 },
  ];
 const gridStyle = { minHeight: 550 }
  return (
    <>
      <ReactDataGrid
        idProperty="id"
        columns={columns}
        style={gridStyle}
        dataSource={dataSource}
      />
    </>
  );
};

export default Main;
```

What you did: Im trying to use library on next js

What happened: Getting Cannot read properties of null (reading 'length')

**Comment by pehcst** [comment]
i have the same problem

**Comment by chiester** [comment]
+1

**Comment by alabhya-ck** [comment]
I am getting `TypeError: Cannot read properties of undefined (reading 'useNativeFlex')`

**Comment by tifosiblack** [comment]
Compatibility with React 18.x, Duplicate of #263 .

**Comment by inovua-admin** [comment]
Fixed in version `5.0.0`.
